sdk/storage/ustorageclient/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 3.2)
+
+project(azure_iot_ustorageclient)
+
+set(AZURE_IOT_USTORAGECLIENT_VERSION 1.0.1)
```

Comment:
you can set this in the project command
```
project(azure_iot_ustorage_client LANGUAGES C VERSION 1.0.1)
```
This sets PROJECT_VERSION, and some other variables, see https://cmake.org/cmake/help/latest/command/project.html

Using this means that other cmake commands that need the version of the project "just work"